core/src/main/java/com/backblaze/b2/json/B2JsonBoundedByteArrayOutputStream.java

```diff
@@ -13,45 +15,51 @@
  * A B2Json implementation of OutputStream that
  * - stores stream content in a byte array
  * - expands capacity when needed (typically doubles)
- * - has a max capacity threshold
- * - throws IOException if the threshold is reached
+ * - has an upper bound limit on output array size (max capacity)
+ * - throws IOException if the max capacity threshold is crossed
  *
  * THREAD-SAFE
  */
 public class B2JsonBoundedByteArrayOutputStream extends OutputStream {
```

Comment:
We'd expose B2JsonBoundedByteArrayOutputStream as public so bzmono can replace standard ByteArrayOutputStream with this type of OutputStream as parameters to some B2Json APIs (to catch the intended IOException). For instance, B2Json.toJson(Object obj, B2JsonOptions options, OutputStream out).